cristin-person/src/main/java/no/unit/nva/cristin/person/update/CristinPersonPatchJsonCreator.java

```diff
@@ -0,0 +1,85 @@
+package no.unit.nva.cristin.person.update;
+
+import static no.unit.nva.cristin.model.JsonPropertyNames.ID;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.FIRST_NAME;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.LAST_NAME;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.ORCID;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.PREFERRED_FIRST_NAME;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.PREFERRED_LAST_NAME;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.RESERVED;
+import org.json.JSONObject;
+
+public class CristinPersonPatchJsonCreator {
+
+    public static final String CRISTIN_FIRST_NAME = "first_name";
+    public static final String CRISTIN_SURNAME = "surname";
+    public static final String CRISTIN_FIRST_NAME_PREFERRED = "first_name_preferred";
+    public static final String CRISTIN_SURNAME_PREFERRED = "surname_preferred";
+
+    private final transient JSONObject input;
```

Comment:
See com.fasterxml.jackson.databind.node.ObjectNode, since we can avoid using org.json.JSONObject